### ISSUE TITLE
Flake8 rule 'E901' reported an invalid point

### ISSUE THREAD

**Issue by brezniczky** [AtomLinter/linter-flake8#405]
Flake8 reported an invalid point for the rule `E901`, with the messge `SyntaxError: invalid syntax`.

```
#!/usr/bin/python

print("hello")

def x():
    pass

def y():
    a = 
    xx + 5
    pass
```

Debug information:
Atom version: 1.14.4
Flake8 version: `2.5.5 (pep8: 1.5.7, mccabe: 0.2.1, pyflakes: 0.8.1, hacking.core: 0.0.1, ProxyChecker: 0.0.1, flake8-docstrings: 1.0.3, pydocstyle: 2.0.0) CPython 2.7.12 on Linux`

**Comment by Arcanemagus** [comment]
Please update your `flake8` version (`pip install -U flake8`). This particular error should be fixed in `flake8>=3.2.1`.

**Comment by brezniczky** [comment]
Thank you, you are likely correct. The update confirms I had flake8 2.5.5 installed, afterwards it seems I am unable to trigger the exception.
